lib/stan.js

```diff
@@ -1049,6 +1049,10 @@ function SubscriptionOptions (durableName, maxInFlight, ackWait, startPosition,
   this.manualAcks = manualAcks
 }
 
+exports.NewSubscriptionOptions = function () {
+  return new SubscriptionOptions()
```

Comment:
What will that return, a subscription options object "not initialized"? As of now, and based on the example [here](https://github.com/nats-io/stan.js/blob/master/examples/stan-sub.js#L42), the way you pass option to stan.subscribe() is to call stan.subscriptionOptions() which will initialize the options with some defaults. If no option is passed to subscribe(), it does internally call stan.subscriptionOptions().

I wonder if this new exported SubscriptionOptions() will then not cause issues since you will potentially pass non-initialized options that will cause the server to reject. For instance, AckWait, MaxInflight need to be specified, etc..

Reply:
it won't - to set options now you simply call a function which creates an instance of SubscriptionOptions the same way:

https://github.com/nats-io/stan.js/pull/172/files#diff-3f142cab90985524df39af9db2ed01e0R1060

Reply:
ok, so the defaults are still set because of this: `this.maxInFlight = maxInFlight || DEFAULT_MAX_IN_FLIGHT` (same for ackwait).

Reply:
correct.